test/fixtures/poro.rb

```diff
@@ -146,6 +146,9 @@ class CustomBlogSerializer < ActiveModel::Serializer
   attribute :special_attribute
   has_many :articles
 end
+class ExternalBlog < Blog
+  attributes :external_id
+end
```

Comment:
This can go in the `test/serializers/associations_test.rb`.  (This file is big enough :) )